lib/getMimeType.js

```diff
@@ -0,0 +1,11 @@
+var MIME_TYPES = ['video/webm;codecs=vp9,opus', 'video/webm;codecs=vp8,opus']
```

Comment:
maybe you prefer lib/utils.js to put this?

Reply:
@soyuka it's probably fine for the moment. I can modularize things later on if files become too hectic.

Reply:
@soyuka ah, I see you've already done it. even better! thank you ^__^